site/_posts/2019-10-09-Custom-E2E-Image.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Simple Approaches to Customizing the Kubernetes E2E Tests
+image: /img/vmware-tanzu.svg
+excerpt: Two approaches to running the E2E tests with custom options to support your workflow.
+author_name: John Schnake
+author_url: https://github.com/johnschnake
+# author_avatar: https://placehold.it/64x64
+categories: [kubernetes, sonobuoy, conformance]
+# Tag should match author to drive author pages
+tags: ['Sonobuoy Team']
+---
+
+# Simple Approaches to Customizing the Kubernetes E2E Tests
+
+Sonobuoy can be used to run any set of custom tests and data-gathering logic, but the most common use case continues to be running the Kubernetes E2E tests.
+
+The [E2E test image](https://github.com/kubernetes/kubernetes/tree/master/cluster/images/conformance) is maintained by the Kubernetes community and is automatically generated for each release of Kubernetes. The test image bundles:
+The binary, which contains the test logic
```

Comment:
These aren't appearing on separate lines on the rendered page. Perhaps they should be bullet points?